### PR TITLE
Minor Introduction, Tools and SmartApps updates.

### PR DESCRIPTION
**Pull request by mrobinet**

### CODE REVIEW

File: introduction/smartthings-architecture.rst [hunk 1]
```diff
@@ -109,7 +109,7 @@ world. There's a huge variety in the devices you can use, some created
 by SmartThings but most are not.
 
 SmartThings Devices
-+++++++++++++++++++
+~~~~~~~~~~~~~~~~~~~
```
Comment (mrobinet) [inline introduction/smartthings-architecture.rst]: A bunch of these bullet points were missing from the navigation.

File: smartapp-developers-guide/preferences-and-settings.rst [hunk 2]
```diff
@@ -216,7 +216,7 @@ uninstall button on the first and third pages:
 ====================================================    ====================================================    ====================================================
 Page 1                                                  Page 2                                                  Page 3
 ====================================================    ====================================================    ====================================================
-.. image:: ../img/smartapps/multiple-pages-page1.png    .. image:: ../img/smartapps/multiple-pages-page2.png    .. image:: ../img/smartapps/multiple-pages-page1.png
+.. image:: ../img/smartapps/multiple-pages-page1.png    .. image:: ../img/smartapps/multiple-pages-page2.png    .. image:: ../img/smartapps/multiple-pages-page3.png
```
Comment (mrobinet) [inline smartapp-developers-guide/preferences-and-settings.rst]: Showing the wrong image for step 3. The step 3 image [exists](/SmartThingsCommunity/Documentation/blob/master/img/smartapps/multiple-pages-page3.png) in the repository, but [results in a 404](http://docs.smartthings.com/en/latest/_images/multiple-pages-page3.png) on the docs website. Not sure if more is needed here?

File: smartapp-developers-guide/preferences-and-settings.rst [hunk 3]
```diff
@@ -278,7 +278,7 @@ Example:
 
     preferences {
         section("paragraph") {
-            icon(title: "required:true",
+            icon(title: "required is true",
```
Comment (mrobinet) [inline smartapp-developers-guide/preferences-and-settings.rst]: Updated the title to match the [related image](/SmartThingsCommunity/Documentation/blob/master/img/smartapps/prefs-icon.png).

**Comment by jimmyjames** [comment]
Good catches. Because we recently updated the intro in a larger change set, I just created a new PR for the preferences fixes: https://github.com/SmartThingsCommunity/Documentation/pull/176

Closing this PR since the changes in this PR are now made in other PRs.

**Pull request closed without merging**